recipes-support/sysctl-hang-crash-helper/sysctl-hang-crash-helper_0.1.bb

```diff
@@ -0,0 +1,15 @@
+SUMMARY = "sysctl hang/crash helper settings"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+inherit allarch
+
+SRC_URI = "file://sysctl-panic.conf"
+
+S = "${WORKDIR}"
+
+PACKAGE_ARCH = "${MACHINE_ARCH}"
+
+do_install () {
+	install -m 0644 ${WORKDIR}/sysctl-panic.conf ${IMAGE_ROOTFS}${exec_prefix}/lib/sysctl.d/60-panic.conf
```

Comment:
Better to install the file at '/usr/lib/sysctl.d' (${D}${libdir}/sysctl.d...)

Make sure to create that folder first as well.